pkg/sql/schemachanger/scplan/internal/rules/current/helpers.go

```diff
@@ -218,6 +218,16 @@ func isConstraint(e scpb.Element) bool {
 	return isIndex(e) || isNonIndexBackedConstraint(e)
 }
 
+func isCheckConstraint(e scpb.Element) bool {
+	switch e.(type) {
+	case *scpb.CheckConstraint:
+		return true
+	case *scpb.CheckConstraintUnvalidated:
+		return true
```

Comment:
nit: we can do `case *scpb.CheckConstraint, *scpb.CheckConstraintUnvalidated: ...`